run.py

```diff
@@ -0,0 +1,45 @@
+from mpi4py import MPI
+import numpy as np
+import adios2
+
+import src.convert_stl as convert_stl
+import src.embed_stl as embed_stl
+
+def run(stl_file):
+
+    comm = MPI.COMM_WORLD
+    rank = comm.Get_rank()
+    size = comm.Get_size()
+        
+    # Convert STL to voxel array
+    voxels = convert_stl.convert(stl_file)
+
+    print(f"Model dimensions: {voxels.L}")
+    print(f"Model scale: {voxels.scale}")
```

Comment:
What does model mean here? is that the stl surface? I think 'model' can be confusing because 'model scale' also means a scaled down version of a physical thing, and it is used a lot in CFD.

Something like 'object' might be easier to understand